any_parser/batch_parser.py

```diff
@@ -0,0 +1,113 @@
+"""Batch parser implementation."""
+
+from typing import List, Optional
+
+import requests
+from pydantic import BaseModel, Field
+
+from any_parser.base_parser import BaseParser
+
+TIMEOUT = 60
+
+
+class UploadResponse(BaseModel):
+    fileName: str
+    requestId: str
+    requestStatus: str
+
+
+class UsageResponse(BaseModel):
+    pageLimit: int
+    pageRemaining: int
+
+
+class FileStatusResponse(BaseModel):
+    fileName: str
+    fileType: str
+    requestId: str
+    requestStatus: str
+    uploadTime: str
+    completionTime: Optional[str] = None
+    result: Optional[List[str]] = Field(default_factory=list)
+    error: Optional[List[str]] = Field(default_factory=list)
```

Comment:
nit: add a docstring and describe what each attribute means to ease the review process.